### ISSUE TITLE
prefer base os no flatpaks,,,,

### ISSUE THREAD

**Issue by outbackdingo** [ublue-os/cosmic#70]
not sure i agree with the latest change to force flatpaks on people, i dont use them, cant use them not allowed to use them, therefor no i must fork and maintain my own copy... great job... until you added flatpaks :) users should build custom images from your base, not have to rip out a pile of things that are not needed.

**Comment by castrojo** [comment]
If flatpaks are a problem for you then none of our images are for you.

**Comment by outbackdingo** [comment]
> Rather than forking the repo, which it appears you have done, you should instead extend the image as you would with any other Docker image. You then can build your own ISO with or without Flatpaks by simply changing the GitHub Actions inputs.

Yes, i found your image, which was perfect for a lean base install of cosmic... i was happy that it was minimal. I did fork it so i could extend it, and create a base image specific to my needs adding various packages like zsh, mc and a few others, so i could reimage multiple laptops i own. i did then have to install some packages like vivaldi slack and others via rpm-ostree install as you mention. my gopal was an iso that i could image multiple systems i use with a lean base. I did achieve that with your cosmic image. i only forked to make customizations i required to the base, youve done great work, im simply extending it for my purposes.
